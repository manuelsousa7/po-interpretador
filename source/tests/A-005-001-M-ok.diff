*** tests/A-005-001-M-ok.out	Thu Nov 17 14:55:35 2016
--- tests/A-005-001-M-ok.outhyp	Fri Nov 18 12:00:52 2016
***************
*** 7,13 ****
  6 - Escrever Programa
  7 - Manipular Programa
  0 - Sair
! Escolha uma opção: Nome do program: Nome do ficheiro do program: Menu Principal
  1 - Criar
  2 - Abrir
  3 - Guardar
--- 7,13 ----
  6 - Escrever Programa
  7 - Manipular Programa
  0 - Sair
! Escolha uma opção: Nome do program: Indique o nome do ficheiro : Menu Principal
  1 - Criar
  2 - Abrir
  3 - Guardar
