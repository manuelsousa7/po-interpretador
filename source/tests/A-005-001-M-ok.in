6
import
pp
0
