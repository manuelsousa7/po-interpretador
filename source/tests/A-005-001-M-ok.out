Menu Principal
1 - Criar
2 - Abrir
3 - Guardar
4 - Criar Programa
5 - Ler Programa
6 - Escrever Programa
7 - Manipular Programa
0 - Sair
Escolha uma opção: Nome do program: Nome do ficheiro do program: Menu Principal
1 - Criar
2 - Abrir
3 - Guardar
4 - Criar Programa
5 - Ler Programa
6 - Escrever Programa
7 - Manipular Programa
0 - Sair
Escolha uma opção: 